### PR TITLE
Docs: Update auditing docs with a basic authentication example

### PR DESCRIPTION
**Pull request by gamab**
**What is this feature?**

Update auditing docs to show an example of how to export logs to a loki instance protected by basic authentication.

Add cloud guidance to set this up.

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Review by stevendungan — COMMENTED** [review]
@gamab I think we can add the following:

"To enable sending Grafana Cloud audit logs to your Grafana Cloud Logs instance, please [file a support ticket](https://grafana.com/profile/org/tickets/new). Note that standard ingest and retention rates apply for ingesting these audit logs."

I think the best place to put this is right below line 20, as an admonition note.

**Review by stevendungan — COMMENTED** [review]
LGTM!

**Review by Jguer — APPROVED** [review]

**Review by mgyongyosi — APPROVED** [review]

**Review by IevaVasiljeva — APPROVED** [review]

File: docs/sources/setup-grafana/configure-security/audit-grafana.md [hunk 1]
```diff
@@ -441,6 +445,15 @@ If you have multiple Grafana instances sending logs to the same Loki service or
 - **grafana_instance** - Application URL.
 - **kind** - `auditing`
 
+When basic authentication is needed to ingest logs in your loki instance, you can specify credentials in the URL field. Here is an example:
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-security/audit-grafana.md:448]:
```suggestion
When basic authentication is needed to ingest logs in your Loki instance, you can specify credentials in the URL field. For example:
```

**Review by chri2547 — APPROVED** [review]
Small wording suggestion for you. Thanks!

**Review by cinaglia — APPROVED** [review]

**Review by buccarel — APPROVED** [review]

**Pull request merged**